src/views/Register.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div style="padding-top: 100px;">
```

Comment:
Avoid using inline styles if not required.

Reply:
ok